cloud/pkg/synccontroller/objectsync.go

```diff
@@ -44,6 +44,7 @@ func (sctl *SyncController) manageObject(sync *v1alpha1.ObjectSync) {
 		newObject.SetNamespace(sync.Namespace)
 		newObject.SetName(sync.Spec.ObjectName)
 		newObject.SetUID(types.UID(getObjectUID(sync.Name)))
+		newObject.SetKind(resourceType)
```

Comment:
It'll send a delete event to edge, could you plesae check that now the error log in edge is only related to the delete event?

Reply:
Yes, I checked the log of the edge, all delete events related.